src/viewport.js

```diff
@@ -139,7 +143,14 @@ export class Viewport {
    * @return {!{width: number, height: number}}
    */
   getSize() {
-    return this.binding_.getSize();
+    var size = this.binding_.getSize();
```

Comment:
That way the line will be clear: Viewport does caching and Binding does not - just translates API directly.